components/events/AccordionSection.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <section
+    class="accordion__section"
+  >
```

Comment:
Nice to have one-liner:

```suggestion
  <section class="accordion__section">
```